offscreen-canvas/the-offscreen-canvas/2d.canvas.readonly.html

```diff
@@ -16,9 +16,10 @@
 var offscreenCanvas = new OffscreenCanvas(100, 50);
 var ctx = offscreenCanvas.getContext('2d');
 
+var offscreenCanvas2 = new OffscreenCanvas(100, 50);
```

Comment:
The offscreenCanvas2 is used in this test after the change.

I search for "offscreenCanvas2" in the test script, and I found that it is used in all the cases.